Source/Documentation/Manual/physics.rst

```diff
@@ -2846,7 +2844,7 @@ where
 
 .. math::
 
-    A=\frac{0.00003318\cdot\rho\cdot F_t}{(1-F_{tr}/F_t)^2},
+    A=\frac{0.00003318\cdot rho\cdot F_t}{(1-F_{tr}/F_t)^2},
```

Comment:
```suggestion
    A=\frac{0.00003318\cdot\rho\cdot F_t}{(1-F_{tr}/F_t)^2},
```
I suggest undoing this. `\rho`, or ρ, is the correct symbol for the density of air.

Reply:
Hi Ryan,

Thanks for this. I couldn't see how to do it.

Please approve,

Chris